network_terminal_input.py

```diff
@@ -0,0 +1,12 @@
+from gamebook.game_input import Input
+from network_terminal_output import NetworkTerminalOutput
+
+
+class NetworkTerminalInput(Input):
+    def __init__(self, client):
+        self.client = client
+        self.output_instance = NetworkTerminalOutput(client)
+
+    def input(self, prompt):
+        self.output_instance.output(bytes(prompt, 'utf-8'))
```

Comment:
איך הלקוח יבין שהוא צריך להפעיל את ה
input?
יעני אין פה הבדל בין זה לבין הOutput

Reply:
i couldn't think of a better way to build this class...
do u have a idea? i couldn't think of a way of improving this :disappointed:

Reply:
Yeah, send a message object (with json - because you need metadata, right?). It could will look something like:
```
{"type": "input", "message": "Please enter an input"}
```
Output could be something similar but a different type.
Then the client can send:
```
{"type": "input-result", "data": "x"}
```

Reply:
message -> data.